bn-models/src/models/_common/assessment-answer-set.ts

```diff
@@ -0,0 +1,16 @@
+import { Base } from '../_base/index';
+
+export const AssessmentAnswerSetTypeName = 'assessment-answer-set';
+
+export interface AssessmentAnswerSet extends Base {
```

Comment:
Just thinking should this extend base-lookup instead of base? Seems like this is lookup info also you'll automatically get the isWcgop and isAshop properties. Same for assessmentQuestion

Reply:
yeah, I went back and forth on whether these were truly lookups.  Since you're leaning that way, I've moved them to lookups and made them inherit from BaseLookup.